lib/backend_common/tests/conftest.py

```diff
@@ -13,31 +16,40 @@ def app():
     # Use unique auth instance
     config = {
         'DEBUG': True,
+        'OIDC_CLIENT_SECRETS': join(dirname(__file__), 'client_secrets.json'),
+        'OIDC_RESOURCE_SERVER_ONLY': True
     }
-    app = create_app('test', extensions=[auth], config=config)
+
+    app = create_app('test', extensions=[auth, auth0], config=config)
 
     @app.route('/')
     def index():
         return app.response_class('OK')
 
-    @app.route('/test-login')
+    @app.route('/test-auth-login')
     @auth.auth.require_login
     def logged_in():
         data = {
             'auth': True,
             'user': current_user.get_id(),
-            'scopes': current_user.permissions,
+            # permissions is a set, not serializable
+            'scopes': list(current_user.permissions),
         }
         return jsonify(data)
 
-    @app.route('/test-scopes')
+    @app.route('/test-auth-scopes')
     @auth.auth.require_scopes([
         ['project/test/A', 'project/test/B'],
         ['project/test-admin/*'],
     ])
     def scopes():
         return app.response_class('Your scopes are ok.')
 
+    @app.route('/test-auth0-userinfo')
+    @auth0.accept_token()
```

Comment:
It's an analog to flask-oidc's `accept_token` decorator. flask-oidc has its own `require_login` intended for interactive use, and `accept_token` is for APIs.

I would like to implement a group filter later on, yes, unless we swap to an auth0 proxy as one of the systems team have mentioned as an option.